### DISCUSSION TITLE
Code Challenge Early Submission Closed

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/GSOC#27] [Announcements]
Early Code Challenge Submission for GSOC 2023 is closed. If you submitted, we'll try to get some feedback to individual participants soon, as well as some general advice based upon observations from the submissions. Please include your code challenge with your proposal.

**Comment by himanshu1409** [comment]
Hey @birm ,could you please open Code challenge for just one day as I had no idea and it was nowhere mentioned that it was going to be closed. I had already made it but forgot to upload it. It's a humble request from my side.This is the GitHub link to it [https://github.com/himanshu1409/Major-Project-2](url)

**Reply by ShivanshAr97** [reply]
Yes that would be helpful

**Reply by ayush-dudhani** [reply]
Yes please @birm open the form for a day, and announce regarding that in slack community please!
